### PR TITLE
Yuwei/bug fix 202302

### PR DESCRIPTION
**Pull request by ivywei0125**
What does this implement/fix? Explain your changes.
---------------------------------------------------
bug fix 

Does this close any currently open issues?
------------------------------------------

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/AzureMonitorView.java [hunk 1]
```diff
@@ -74,9 +75,16 @@ public JPanel getContentPanel() {
...
     }
 
-    private void updateWorkspaceNameLabel(String name) {
-        this.workspaceName.setText(name);
-        this.workspaceName.setToolTipText(name);
+    private void updateWorkspaceNameLabel() {
+        if (Objects.nonNull(selectedWorkspace)) {
+            this.workspaceName.setText(selectedWorkspace.getName());
+            this.workspaceName.setToolTipText(selectedWorkspace.getName());
+            this.workspaceName.setIcon(null);
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-monitor/src/main/java/com/microsoft/azure/toolkit/intellij/monitor/view/AzureMonitorView.java]: use workspace icon

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-eventhubs/src/main/java/com/microsoft/azure/toolkit/intellij/eventhubs/IntellijEventHubsActionsContributor.java [hunk 2]
```diff
@@ -32,19 +32,28 @@ public void registerHandlers(AzureActionManager am) {
 
     private void registerActiveActionHandler(AzureActionManager am) {
         final BiPredicate<EventHubsInstance, AnActionEvent> condition = (r, e) -> true;
-        final BiConsumer<EventHubsInstance, AnActionEvent> handler = (c, e) -> c.active();
+        final BiConsumer<EventHubsInstance, AnActionEvent> handler = (c, e) -> {
+            c.active();
+            EventHubsToolWindowManager.getInstance().updateEventHubsPanelView(e.getProject(), c);
```
Comment (ivywei0125) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-eventhubs/src/main/java/com/microsoft/azure/toolkit/intellij/eventhubs/IntellijEventHubsActionsContributor.java]: should listen to event bus

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-eventhubs/src/main/java/com/microsoft/azure/toolkit/intellij/eventhubs/view/EventHubsSendListenPanel.java [hunk 3]
```diff
@@ -93,6 +89,13 @@ private void init() {
                 new GridConstraints(0, 0, 1, 1, 0, GridConstraints.ALIGN_FILL,
                         3, 3, null, null, null, 0));
         this.sendMessageBtn.setEnabled(this.instance.isActive());
+        AzureEventBus.on("resource.status_changed.resource", new AzureEventBus.EventListener((azureEvent) -> {
+            final String type = azureEvent.getType();
+            final Object source = azureEvent.getSource();
+            if (source instanceof EventHubsInstance && ((EventHubsInstance) source).getId().equals(this.instance.getId())) {
+                this.sendMessageBtn.setEnabled(((EventHubsInstance) source).isActive());
+            }
+        }));
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-eventhubs/src/main/java/com/microsoft/azure/toolkit/intellij/eventhubs/view/EventHubsSendListenPanel.java]: should `off` event when closed.

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-eventhubs/src/main/java/com/microsoft/azure/toolkit/intellij/eventhubs/IntellijEventHubsActionsContributor.java [hunk 4]
```diff
@@ -32,28 +32,19 @@ public void registerHandlers(AzureActionManager am) {
 
     private void registerActiveActionHandler(AzureActionManager am) {
         final BiPredicate<EventHubsInstance, AnActionEvent> condition = (r, e) -> true;
-        final BiConsumer<EventHubsInstance, AnActionEvent> handler = (c, e) -> {
-            c.active();
-            EventHubsToolWindowManager.getInstance().updateEventHubsPanelView(e.getProject(), c);
-        };
+        final BiConsumer<EventHubsInstance, AnActionEvent> handler = (c, e) -> c.active();
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-eventhubs/src/main/java/com/microsoft/azure/toolkit/intellij/eventhubs/IntellijEventHubsActionsContributor.java]: active -> activate

**Review by wangmingliang-ms — APPROVED** [review]
LGTM

**Pull request merged**